memgpt/cli/cli.py

```diff
@@ -106,6 +106,7 @@ def run(
     model_endpoint_type: str = typer.Option(None, help="Specify the LLM model endpoint type"),
     context_window: int = typer.Option(None, help="The context window of the LLM you are using (e.g. 8k for most Mistral 7B variants)"),
     # other
+    server: bool = typer.Option(False, "--server", help="Start MemGPT as a Web Server"),
```

Comment:
Do we need this? I haven't look at the chain of execution but I started with `memgpt server` and it worked - does this code get touched?